cobigen-templates/crud-ionic-client-app/src/main/resources/context.xml

```diff
@@ -1,16 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<contextConfiguration xmlns="http://capgemini.com/devonfw/cobigen/ContextConfiguration" version="2.2">
-
-
-
+<contextConfiguration xmlns="http://capgemini.com/devonfw/cobigen/ContextConfiguration" version="2.3">
```

Comment:
The latest version for the context.xml will be 3.0, right? So the version does not match. Also for the other context configurations